medic/medic-downloadlogs.js

```diff
@@ -0,0 +1,80 @@
+var path = require('path'),
+    fs = require('fs'),
+    https = require('https'),
+    optimist = require('optimist');
+    q = require('Q'),
+    mkdirp = require('mkdirp')
+    util = require('util');
+
+var SERVER = "https://ci.apache.org";
+var BUILDERS = ["cordova-windows-store8.1", "cordova-ios", "cordova-windows-phone8.1", "cordova-android-osx","cordova-android-win"];
+//var BUILDERS = ["cordova-windows-store8.1"];
```

Comment:
Removed!

---
In reply to: [59606834](https://github.com/apache/cordova-medic/pull/87#discussion_r59606834) [](ancestors = 59606834)